itou/www/siaes_views/views.py

```diff
@@ -56,107 +61,240 @@ def job_description_card(request, job_description_id, template_name="siaes/job_d
         .exclude(id=job_description_id)
         .order_by("-updated_at", "-created_at")
     )
+    breadcrumbs = {
+        "Métiers et recrutements": reverse("siaes_views:job_description_list"),
+        "Détail du poste": reverse(
+            "siaes_views:job_description_card",
+            kwargs={
+                "job_description_id": job_description_id,
+            },
+        ),
+    }
     context = {
         "job": job_description,
         "siae": siae,
         "others_active_jobs": others_active_jobs,
         "back_url": back_url,
+        "breadcrumbs": breadcrumbs,
     }
     return render(request, template_name, context)
 
 
 @login_required
-def configure_jobs(request, template_name="siaes/configure_jobs.html"):
+def card_search_preview(request, template_name="siaes/includes/_card_siae.html"):
     """
-    Configure an SIAE's jobs.
-
-    Time was limited during the prototyping phase and this view is based on
-    JavaScript to generate a dynamic form. No proper Django form is used.
+    SIAE's card (or "Fiche" in French) search preview.
+    Return only html of card search preview without global template (header, footer, ...)
```

Comment:
comme dit ailleurs, je pense qu'il est inutile de mettre un commentaire quand on décrit "ce que fait" le code. Les commentaires devraient documenter ce que le code ne montre pas déjà (le contexte métier, les choix originels, etc)

https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/

Reply:
C'est un vieux commentaire non ? J'ai du le copier/coller. Mais je suis d'accord